dgraph-modus-example/frontend/app/[id]/page.tsx

```diff
@@ -0,0 +1,122 @@
+import { Suspense } from "react"
+import { fetchMovieDetailsAndRecommendations } from "@/app/actions"
+import Link from "next/link"
+
+function Loading() {
```

Comment:
For some reason I'm not seeing this loading state when running locally - only the link back to Movies. 
Any idea why this doesn't show up?
<img width="1256" alt="Screenshot 2024-12-20 at 11 42 54 AM" src="https://github.com/user-attachments/assets/ff44a09e-481d-4e11-9daa-c1d6240382de" />

Reply:
Fixed in 0549c14a5e11d46f723f41c004ecf4d17b8102ee